src/AutoRest.CSharp/Common/Generation/Writers/JsonCodeWriterExtensions.cs

```diff
@@ -141,7 +142,9 @@ public static void ToSerializeCall(this CodeWriter writer, JsonSerialization ser
                                  frameworkType == typeof(DateTime) ||
                                  frameworkType == typeof(TimeSpan))
                         {
-                            if (valueSerialization.Format == SerializationFormat.DateTime_Unix)
+                            if (valueSerialization.Format == SerializationFormat.DateTime_Unix ||
```

Comment:
maybe we can use pattern matching to simplify the if check here.

Reply:
Thanks @archerzz Good suggestion, but now we keep to use this syntax.